org.eclipse.xtext.common.types.eclipse.tests/META-INF/MANIFEST.MF

```diff
@@ -17,7 +17,6 @@ Require-Bundle: org.eclipse.xtext.junit4,
  org.eclipse.jdt.launching;bundle-version="3.9.100",
  org.eclipse.xtext.common.types.ui,
  org.eclipse.emf.mwe2.launch;bundle-version="2.9.1";resolution:=optional;x-installation:=greedy,
```

Comment:
mwe2 launch - shouldn't this be moved to the additional.bundles section, too?

Reply:
might be some tests that use some classes transitively at runtime.
need to check that